pkg/agent/config_test.go

```diff
@@ -178,7 +178,7 @@ func Test_ValidateAndCombineConfig(t *testing.T) {
 
 		// The log line printed by pflag is not captured by the log recorder.
 		assert.Equal(t, testutil.Undent(`
-			INFO Using the Jetstack Secure OAuth auth mode since --credentials-file was specified without --venafi-cloud.
+			INFO Authentication mode mode="Jetstack Secure OAuth" reason="--credentials-file was specified without --venafi-cloud"
```

Comment:
Another drive-by fix. This test was introduced in #628 but #625 which altered the logging format (and which was merged afterwards) had not been rebased and did not alter the expected log output in this test.

Reply:
The tests are not being run on the master branch and the green badge on the README file does not reflect the status of the current branch:
 * https://github.com/jetstack/jetstack-secure/actions/workflows/tests.yaml?query=branch%3Amaster
 * https://github.com/jetstack/jetstack-secure/blob/master/README.md